### ISSUE TITLE
Fix pub package description and authors for pub.dartlang.org

### ISSUE THREAD

**Issue by ErikGrimes** [dart-lang/polymer_elements#107]
The description and authors displayed by pub.dartlang.org in package listings and search results don't match the current pubspec for the package. For me, it shows Gunter and I as authors with the description:

```
Deprecated and unsupported. A Dart port of polymer-elements (https://github.com/Polymer/polymer-elements).
```

**Comment by jakemac53** [comment]
Weird... I see the right thing?

**Comment by jakemac53** [comment]
Maybe try refreshing a few times

**Comment by ErikGrimes** [comment]
I've tried different browsers (Chrome, Firefox, Safari), different devices (including coworkers phones and computers I've never logged into), different networks (at&t, timewarner, tmobile).  They're all showing the same erroneous information.

**Comment by ErikGrimes** [comment]
Here's a screenshot taken from Edge on Win 10 via Browserstack.

![pub-listing](https://cloud.githubusercontent.com/assets/1533326/10890010/f1ececda-815a-11e5-8bcf-2f07128d776e.jpg)

**Comment by zoechi** [comment]
This view shows the information of the last release (non-pre-release)

**Comment by ErikGrimes** [comment]
Ahh, makes sense.  It's a bit disconcerting though.  Hopefully its not discouraging anyone from playing with release candidates.  

@jakemac53 Feel free to close this if you like.

**Comment by jakemac53** [comment]
Weird that the search view doesn't show the most recent version? Maybe it has something to do with the `-rc.*`?

**Comment by zoechi** [comment]
Yup, that was what I mean with "non-pre-release". The main page doesn't get updated for `-xxx` version.

**Comment by zoechi** [comment]
also like `pub get`/`pub upgrade` by default ignore `-xxx` versions if your version constraint doesn't have a `-` after the patch version.
